RetroBar/Languages/română.xaml

```diff
@@ -3,71 +3,71 @@
-    <s:String x:Key="show_multi_mon">Arată pe _ecrane multiple</s:String>
-    <s:String x:Key="show_quick_launch">Arată _Meniul de Lansare Rapidă</s:String>
-    <s:String x:Key="select_location">_Selectează o locație...</s:String>
+    <s:String x:Key="allow_font_smoothing">_Activați netezirea fontului</s:String>
+    <s:String x:Key="collapse_tray_icons">Ascundeți pictogramele din bara de notificări</s:String>
+    <s:String x:Key="customize">Personalizați...</s:String>
+    <s:String x:Key="show_clock">Afișați _ceasul</s:String>
+    <s:String x:Key="show_multi_mon">Afisați pe _ecrane multiple</s:String>
+    <s:String x:Key="show_quick_launch">Afișați _Meniul de Lansare Rapidă</s:String>
+    <s:String x:Key="select_location">_Selectați o locație...</s:String>
     <s:String x:Key="quick_launch_folder">Meniul de Lansare Rapidă - Selectează un dosar</s:String>
-    <s:String x:Key="use_software_rendering">_Folosește software rendering</s:String>
+    <s:String x:Key="use_software_rendering">_Folosiți software rendering</s:String>
     <s:String x:Key="ok_dialog">OK</s:String>
 
-    <s:String x:Key="customize_notifications">Personalizează Notificările</s:String>
-    <s:String x:Key="customize_notifications_info">RetroBar arată pictograme pentru notificări active și urgente, și le ascunde pe cele inactive. Poți schimba acest comportament pentru următoarele din lista de jos.</s:String>
-    <s:String x:Key="customize_notifications_instruction">Selectează o opțiune, apoi alege comportamentul notificării:</s:String>
+    <s:String x:Key="customize_notifications">Personalizați Notificările</s:String>
+    <s:String x:Key="customize_notifications_info">RetroBar arată pictograme pentru notificări active și urgente, și le ascunde pe cele inactive. Puteți schimba acest comportament pentru următoarele din lista de jos.</s:String>
+    <s:String x:Key="customize_notifications_instruction">Selectați o opțiune, apoi alegeți comportamentul notificării:</s:String>
     <s:String x:Key="hide_when_inactive">Ascunde în caz de inactivitate</s:String>
     <s:String x:Key="always_show">Mereu arată</s:String>
     <s:String x:Key="name_heading">Nume</s:String>
     <s:String x:Key="behavior_heading">Comportament</s:String>
 
     <s:String x:Key="start_text">Start</s:String>
     <s:String x:Key="start_text_xp">start</s:String>
-    <s:String x:Key="start_button_tip_98">Dă click aici să începi.</s:String>
-    <s:String x:Key="start_button_tip">Dă click aici să începi</s:String>
+    <s:String x:Key="start_button_tip_98">Click aici pentru a începe.</s:String>
+    <s:String x:Key="start_button_tip">Click aici pentru a începe</s:String>
     <s:String x:Key="start_button_tip_vista">Start</s:String>
 
-    <s:String x:Key="retrobar_title">Bara de activități RetroBar</s:String>
-    <s:String x:Key="toolbars">_Bare de activități</s:String>
+    <s:String x:Key="retrobar_title">Bara de instrumente RetroBar</s:String>
```

Comment:
This is the title of the program, is displayed when the program is not responding, and refers to the taskbar.
Wouldn't it be "activități" instead of "instrumente"?

Reply:
That is correct. I thought in the English text it was "toolbar".